packages/server/test/unit/config_spec.coffee

```diff
@@ -347,6 +347,15 @@ describe "lib/config", ->
           @setup({video: 42})
           @expectValidationFails("be a boolean")
 
+      context "videoUpload", ->
+        it "passes if a boolean", ->
+          @setup({videoUpload: false})
+          @expectValidationPasses()
+
+        it "fails if not a boolean", ->
```

Comment:
little more truthful: "fails if a number" (it might even pass if it's the number 1?)

Reply:
You're completely right, although I've honestly just copied it over from other tests in order to comply with what they have. I would change it of course if I'm free to do so.